cli/cmd/root.go

```diff
@@ -82,9 +83,23 @@ var rootCmd = &cobra.Command{
 			}
 		}
 
+		if versionMatch, cliVersion, lockedVersion, err := VerifyHappyIsLockedVersion(cmd); err != nil {
+			return errors.Wrap(err, "Unable to verify locked Happy version")
+		} else {
+			if !versionMatch {
+				return errors.New(
+					fmt.Sprintf("Installed Happy version (%s) does not match locked version in .happy/version.lock (%s)", cliVersion, lockedVersion),
+				)
```

Comment:
```suggestion
				return errors.Error("Installed Happy version (%s) does not match locked version in .happy/version.lock (%s)", cliVersion, lockedVersion)
```

Reply:
let's also provide the command to get that particular version (I always forget the brew commands)